__tests__/get.test.js

```diff
@@ -56,4 +56,55 @@ describe('/api', () => {
             })
 
     })
+})
+describe('/api/articles/:article_id', () => {
+    test('to GET an article object containing the following properties: author, title, article_id, body, topic, created_at, votes, article_img_url', () => {
+        return request(app)
+            .get('/api/articles/9')
+            .expect(200)
+            .then((response) => {
+                const articleObject = response.body.article
+                expect(articleObject.hasOwnProperty('author')).toBe(true);
+                expect(articleObject.hasOwnProperty('title')).toBe(true);
+                expect(articleObject.hasOwnProperty('article_id')).toBe(true);
```

Comment:
Here I would like us to specifically test the article_id is 9 - the others we can just check hasOwnProperty, but we want to ensure we are getting the correct article back :)